### PR TITLE
Feat/update book

### LINKED ISSUE #30: Remove bookmark button in Page isn't always displaying the error message

### PR DESCRIPTION
**Pull request by Jesusborjas006**
Change Summary: 
- This will update the pages component by adding an error message when clicking the "Remove Bookmark" button. It's going to let the user know to go to previous page to display a fact. When there are no facts saved the user is then displayed with a message saying that there's no facts! Fixes issue by decrementing page number when removing a fact.

Function/Methods: 
- Inline styling to toggle the error and fact using state.

closes #30

### CODE REVIEW

**Review by BriBourassa — APPROVED** [review]
The book looks great and the buttons are so much better! Love the page count functionality

File: src/components/Book/Book.js [hunk 1]
```diff
@@ -33,15 +32,24 @@ function Book() {
...
+          <button className="save-fact" onClick={() => dispatch(addFact(fact))}>
+            Bookmark Fact
+          </button>
+        )}
+        {savedFacts.includes(fact) && (
+          <button onClick={() => dispatch(removeFact(fact))}>
+            Remove Bookmark
+          </button>
+        )}
+        {savedFacts[0] && (
```
Comment (qrispi) [inline src/components/Book/Book.js:45]: I really need to install Prettier

**Review by jsahim — APPROVED** [review]
Looks great!

File: src/components/Page/Page.js
Comment (qrispi) [inline src/components/Page/Page.js:1]: Man this got so much headier than I was expecting! Great job on the error handling!

**Review by qrispi — APPROVED** [review]
YaY! Book & Page look fantastic

**Pull request merged**